openedx/features/caliper_tracking/settings.py

```diff
@@ -0,0 +1,17 @@
+"""
+All settings for Caliper Tracking app.
+"""
+
+LOGIN_EVENT_EMITTER = 'caliper_tracking.helpers.emit_login_event'
+CALIPER_TRACKING_PROCESSOR = 'openedx.features.caliper_tracking.processor.CaliperProcessor'
```

Comment:
Please manually test all of this by creating a dummy repo of this and then testing through site packages.

Reply:
Please use proper path.

Reply:
we can use proper paths once this is converted into separate app. Then all of this code in `openedx` won't be required. but right now. we have to use these paths.